docs/csharp/versioning.md

```diff
@@ -7,26 +7,22 @@ ms.assetid: aa8732d7-5cd0-46e1-994a-78017f20d861
 
 # Versioning in C\#
 
-In this tutorial you'll learn what versioning means in .NET. You'll also learn the factors to consider when versioning your library as well as upgrading
-to a new version of a library.
+In this tutorial you'll learn what versioning means in .NET. You'll also learn the factors to consider when versioning your library as well as upgrading to a new version of a library.
 
 ## Authoring Libraries
 
-As a developer who has created .NET libraries for public use, you've most likely been in situations where you have
-to roll out new updates. How you go about this process matters a lot as you need to ensure that there's a seamless transition
-of existing code to the new version of your library. Here are several things to consider when creating a new release:
+As a developer who has created .NET libraries for public use, you've most likely been in situations where you have to roll out new updates. How you go about this process matters a lot as you need to ensure that there's a seamless transition of existing code to the new version of your library. Here are several things to consider when creating a new release:
 
 ### Semantic Versioning
 
 [Semantic versioning](https://semver.org/) (SemVer for short) is a naming convention applied to versions of your library to signify specific milestone events.
-Ideally, the version information you give your library should help developers determine the compatibility
-with their projects that make use of older versions of that same library.
+Ideally, the version information you give your library should help developers determine the compatibility with their projects that make use of older versions of that same library.
 
 The most basic approach to SemVer is the 3 component format `MAJOR.MINOR.PATCH`, where:
 
-* `MAJOR` is incremented when you make incompatible API changes
-* `MINOR` is incremented when you add functionality in a backwards-compatible manner
-* `PATCH` is incremented when you make backwards-compatible bug fixes
+- `MAJOR` is incremented when you make incompatible API changes
```

Comment:
Incidentally, `*` and `-` render the same.

Reply:
@rpetrusha, Yes. but I see the `-` used more in the docs, so made this change for more consistency.